Documentation/Update/Index.rst

```diff
@@ -0,0 +1,104 @@
+.. include:: ../Includes.txt
+
+.. highlight:: bash
+
+.. _update:
+
+==================
+TYPO3 Patch Update
+==================
+
+
+Updating major and minor versions is handled in :ref:`upgrade`. A major update
```

Comment:
We say there is no need for `database analyzer`, but add exactly that point afterwards.

I guess we should shorten this sentence down to the minimum and do not mention concrete stuff, which might change in the future.

Reply:
Daniel I think it is worthwhile to mention the difference here. We could say "requires running the Upgrade Wizard and usually the database analyzer". In comparition the patch update only requires the database analyzer in rare cases and if the release notes state so.